pkg/log/fatalHook.go

```diff
@@ -38,16 +39,78 @@ func (f *FatalHook) Fire(entry *logrus.Entry) error {
 	fileName := "errorDetails.json"
 	if details["stepName"] != nil {
 		fileName = fmt.Sprintf("%v_%v", fmt.Sprint(details["stepName"]), fileName)
+		// ToDo: If step is called x times, and it fails multiple times the error is overwritten
 	}
 	filePath := filepath.Join(f.Path, fileName)
+	filePathCPE := filepath.Join(f.Path, ".pipeline", "commonPipelineEnvironment", fileName)
```

Comment:
could the `piperenv` package be used here instead of replicating the cpe logic?

Reply:
Encountered problems with a cycle import inside the fatalhook package. Appreciate any guidance on how to disentangle that correctly.